cmake/printInfo.cmake

```diff
@@ -31,12 +32,16 @@ else()
 endif(NOT DEFINED BUILD_SHARED_LIBS)
 message( STATUS "BUILD_TESTS =           ${BUILD_TESTS}" )
 message( STATUS "ENABLE_CHECK_INTERNAL = ${ENABLE_CHECK_INTERNAL}" )
+message( STATUS "ENABLE_OSS_FUZZ =       ${ENABLE_OSS_FUZZ}" )
+message( STATUS )
 message( STATUS "BUILD_GUI =             ${BUILD_GUI}" )
 message( STATUS "WITH_QCHART =           ${WITH_QCHART}" )
 message( STATUS )
 message( STATUS "HAVE_RULES =            ${HAVE_RULES}" )
+message( STATUS "PCRE_LIBRARY =          ${PCRE_LIBRARY}" )
 message( STATUS )
-message( STATUS "Change a value with: cmake -D<Variable>=<Value>" )
+message( STATUS "USE_Z3 =                ${USE_Z3}" )
+message( STATUS "Z3_LIBRARY =            ${Z3_LIBRARY}" )
```

Comment:
It only makes sense to show status info about the `Z3_LIBRARY`, if `USE_Z3` is defined.
So, I would suggest the following:
```diff
diff --git a/cmake/printInfo.cmake b/cmake/printInfo.cmake
index bbe32b2b8..dc967abe2 100644
--- a/cmake/printInfo.cmake
+++ b/cmake/printInfo.cmake
@@ -41,7 +41,10 @@ message( STATUS "HAVE_RULES =            ${HAVE_RULES}" )
 message( STATUS "PCRE_LIBRARY =          ${PCRE_LIBRARY}" )
 message( STATUS )
 message( STATUS "USE_Z3 =                ${USE_Z3}" )
-message( STATUS "Z3_LIBRARY =            ${Z3_LIBRARY}" )
+if(${USE_Z3})
+    message( STATUS "Z3_LIBRARY =            ${Z3_LIBRARY}" )
+    message( STATUS "Z3_INCLUDE =            ${Z3_INCLUDE}" )
+endif()
 message( STATUS )
 
 if(${ANALYZE_ADDRESS})
```

Reply:
Will do . And as I said - baby steps :)